File: garden_advice.js
function getAdvice() {
    let season = prompt("Enter the season (summer/winter):").toLowerCase().trim();
    let plantType = prompt("Enter the plant type (flower/vegetable):").toLowerCase().trim();

    function getSeasonAdvice(season) {
        if (season === "summer") {
            return "Water your plants regularly and provide some shade.\n";
        } else if (season === "winter") {
            return "Protect your plants from frost with covers.\n";
        } else {
            return "No advice for this season.\n";
        }
    }

    function getPlantAdvice(plantType) {
        if (plantType === "flower") {
            return "Use fertiliser to encourage blooms.";
        } else if (plantType === "vegetable") {
            return "Keep an eye out for pests!";
        } else {
            return "No advice for this type of plant.";
        }
    }

    let advice = getSeasonAdvice(season) + getPlantAdvice(plantType);

    console.log(advice);
    alert(advice); // optional nice touch for user feedback
}

File: garden_advice.py
# Get user input
season = input("Enter the season (summer/winter): ").lower().strip()
plant_type = input("Enter the plant type (flower/vegetable): ").lower().strip()

# Function to handle season advice
def get_season_advice(season):
    if season == "summer":
        return "Water your plants regularly and provide some shade.\n"
    elif season == "winter":
        return "Protect your plants from frost with covers.\n"
    else:
        return "No advice for this season.\n"

# Function to handle plant advice
def get_plant_advice(plant_type):
    if plant_type == "flower":
        return "Use fertilisers to encourage blooms."
    elif plant_type == "vegetable":
        return "Keep an eye out for pests!"
    else:
        return "No advice for this type of plant."

# Combine results
advice = get_season_advice(season) + get_plant_advice(plant_type)

print(advice)
